tools/integration_tests/util/setup/setup.go

```diff
@@ -183,31 +197,24 @@ func ExecuteTest(m *testing.M) (successCode int) {
 	return successCode
 }
 
-func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
+func ExecuteTestForFlagsSet(flags []string, m *testing.M) (successCode int) {
 	var err error
 
-	for i := 0; i < len(flags); i++ {
-		if err = MountGcsfuse(flags[i]); err != nil {
-			LogAndExit(fmt.Sprintf("mountGcsfuse: %v\n", err))
-		}
-
-		// Clean the mountedDirectory before running any tests.
-		os.RemoveAll(mntDir)
-
-		successCode = ExecuteTest(m)
+	// Clean the mountedDirectory before running any tests.
+	os.RemoveAll(mntDir)
 
-		err = UnMount()
-		if err != nil {
-			LogAndExit(fmt.Sprintf("Error in unmounting bucket: %v", err))
-		}
+	successCode = ExecuteTest(m)
```

Comment:
where is this ExecuteTest defined ?

Reply:
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/util/setup/setup.go#L178 In setup.go only